### ISSUE TITLE
Redstone Clock interaction breaks world interaction

### ISSUE THREAD

**Issue by LaVidaYokel** [WesCook/Homestead#42]
Placing or interacting with the Redstone Clock from Vanilla Automation will prevent new blocks from being placed. Relogging or portaling to another dimension fixes problem.

**Comment by WesCook** [comment]
I've seen this reported before, but have had trouble reproducing it consistently.  When I could reproduce it, it seemed to cause a sort of lag or desync with block placement, or picking up unique items.  Once I'd picked up an item, it seemed to be instant again in the future.

Do you have any other details of the behavior of this bug, so I can report it to the relevant mod author?

Thanks!

**Comment by LaVidaYokel** [comment]
What you described is pretty spot-on to what I experienced except it's been consistent for me for each clock I've placed or interacted with.

Here's an example of what happened to me:
I placed a clock then I crafted a lever. I placed the lever and the lever immediately disappears. I check my inventory and there is no lever.  If I open a container or zone to another dimension, when I check my inventory, I find the stick and cobble I used to make the lever still in my inventory.    

Another example:  I interact with a clock's GUI then I place a block.  The block immediately disappears.  The block doesn't reappear in my inventory until after I open a container or I zone. 

In both scenarios, I couldn't place anything in the world until after I zoned or relogged, regardless of whether I opened a container or not.   

I don't know if it's relevant, but I saw a bug report for the clock from 1.12 that was sounded similar. I swear it was on their github but I can't find it now and my search history isn't helping.  If I come across it again, I'll attach it for you.
